packages/circuit-ui/index.ts

```diff
@@ -147,6 +147,10 @@ export { ModalProvider } from './components/ModalContext';
 export type { ModalProviderProps } from './components/ModalContext';
 export { useModal } from './components/Modal';
 export type { ModalProps } from './components/Modal';
+export { default as ListItem } from './components/ListItem';
+export type { ListItemProps } from './components/ListItem';
+export { default as ListItemGroup } from './components/ListItemGroup';
+export type { ListItemGroupProps } from './components/ListItemGroup';
```

Comment:
You probably have a small conflict here with another new component. Should be an easy fix 🙂